contrib/packs/config.yaml

```diff
@@ -1,2 +1,30 @@
 ---
 pack_group: st2packs
+
+repositories:
+  default:
+    protocol: "ssh"
```

Comment:
Would it be too much work to support remote URLs directly? It would be a bit easier for the user.